docs/bigcommerce-for-wordpress/extending-the-plugin/customization-guide.md

```diff
@@ -65,6 +80,145 @@ As WordPress loads, it will first check for a custom template override in your t
+$post_id = $product->post_id(); // the WordPress post ID
+$sku = $product->sku();
+$brand = $product->brand();
+```
+
+Consult the code reference for a full list of methods available on the Product object.
+
+For any data not directly exposed through a dedicated method, call `$product->get_property()` to retrieve the value.
+
+```php
+$weight = $product->get_property( 'weight' );
+$height = $product->get_property( 'height' );
+```
+
+You can retrieve the same properties using the magic `__get()` method on the Product:
+
+```php
+$weight = $product->weight;
+$height = $product->height;
+```
+
+On the client side, the product ID appears in a number of places. Which you might use depends on context. Here are some places to look:
+
+- On an Add to Cart button
+ </br>
+  `var product_id = $('.bc-btn--add_to_cart').attr('data-js')`
+- On the product price
+  </br>
+  `var product_id = $('.bc-product__pricing').attr('data-product-price-id')`
+
+Generally the WordPress plugin works with post IDs, not product IDs, so the latter is rarely needed on the client side.
+
+To retrieve additional information about the product in the browser, there is a REST API endpoint. It's primary purpose is supporting the product block interface in the WordPress admin, but it can be used anywhere to retrieve a small subset of the information about the product. The endpoint is `/wp-json/bigcommerce/v1/products`.
+
+### Variants
+Retrieve a Product object as above. From there, you can retrieve information about variants.
+
+`$variants = $product->variants;`
+
+The returned objects will match the schema from the BigCommerce API.
+
+```php
+$variant_ids = wp_list_pluck( $variants, 'id' );
+foreach ( $variants as $variant ) {
+  $sku = $variant->sku;
+  $inventory = $variant->inventory_level;
+}
+```
+
+On the client side, variant details are available in the product form.
+
+```php
+var variants = JSON.parse($('[data-js="product-variants-object"]').attr('data-variants'));
+var variant_ids = variants.map( variant => variant.variant_id );
+```
+
+The schema does not completely match the API data. It has been adjusted to suit the needs of the product form. Properties include:
+- variant_id
+- price
+- formatted_price
+- sku
+- disabled
+- disabled_message
+- image
+- option_ids
+
+### Channels
+The current channel is available through a Connections object.
+
+```php
+$connections = new \BigCommerce\Taxonomies\Channel\Connections();
+$channel     = $connections->current();
+```
+
+The response is a `WP_Term` object, with meta containing the channel ID:
+
+```php
+$channel_name = $channel->name;
+$channel_id   = get_term_meta( $channel->term_id, \BigCommerce\Taxonomies\Channel\Channel::CHANNEL_ID, true );
+```
+
+The channel ID is not available anywhere on the client side.
+### Customers
+
+A logged out user does not have any customer information. For a logged in user, you can create a Customer object and get information about that customer.

```

Comment:
For a logged in user, you can create a Customer object **to get the customer's information.**